player/screenshot.c

```diff
@@ -64,7 +65,8 @@ void screenshot_init(struct MPContext *mpctx)
     *mpctx->screenshot_ctx = (screenshot_ctx) {
         .mpctx = mpctx,
         .frameno = 1,
-        .log = mp_log_new(mpctx, mpctx->log, "screenshot")
+        .log = mp_log_new(mpctx, mpctx->log, "screenshot"),
+        .sws = mp_sws_alloc(mpctx),
```

Comment:
given that this lives for the entire duration of mpv, maybe worth creating it in the place it's needed?